tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -1112,6 +1112,17 @@ Step #1 might already solve the problem, however if it still persists, execute s
 This issue occurs with new Anaconda installations when `pip` tries to remove `easy-install.pth`. 
 This file is not included in Anaconda packages, which causes the `pip` installation to fail.
 
+#### Cupti_wrapper.cc: Could not find cuptiActivityRegisterCallbacksin libcupti DSO
+
+If running TensorFlow Python script you encounter the error:
```

Comment:
Nit: Replace this with "If, when running a TensorFlow Python script, you encounter the following error:"